.github/workflows/ci.yml

```diff
@@ -110,6 +110,7 @@ jobs:
   # These tests detect if changes to ontologies, libraries, models and BuildingMOTIF
   # affect correct operation of templates, shapes, and validation
   libraries:
+    if: github.event.review.state == 'approved' || github.event.pull_request.merged
```

Comment:
I think you missed adding the event triggers at the top of the file

Reply:
Oh, do I need both? The `library` tests are the longest and I wanted only those to run on approval/merge